admin/src/app/foms/public-notice/public-notice-edit.component.ts

```diff
@@ -9,7 +9,8 @@ import {
 import { RxFormBuilder } from '@rxweb/reactive-form-validators';
 import { User } from "@utility/security/user";
 import { lastValueFrom, map, Subject, switchMap } from 'rxjs';
-import { KeycloakService } from "../../../core/services/keycloak.service";
+// import { KeycloakService } from "../../../core/services/keycloak.service";
+import { CognitoService } from "../../../core/services/cognito.service";
```

Comment:
cleaner to use relative paths for import (@core if I remember correctly).

Reply:
Fixed.